cql3/expr/expression.cc

```diff
@@ -1100,49 +1101,90 @@ std::ostream& operator<<(std::ostream& os, const column_value& cv) {
 }
 
 std::ostream& operator<<(std::ostream& os, const expression& expr) {
+    expression::printer pr {
+        .expr_to_print = expr,
+        .debug_mode = true
+    };
+
+    return os << pr;
+}
+
+std::ostream& operator<<(std::ostream& os, const expression::printer& pr) {
+    // Wraps expression in expression::printer to forward formatting options
+    auto to_printer = [&pr] (const expression& expr) -> expression::printer {
+        return expression::printer {
+            .expr_to_print = expr,
+            .debug_mode = pr.debug_mode
+        };
+    };
+
     expr::visit(overloaded_functor{
             [&] (const constant& v) { os << v.view(); },
```

Comment:
Argh the printer is printing `raw_value_view` here.
I think that's the hex representation of raw bytes :/
This might work for small ints that have the same hex version but other stuff probably breaks here.